packages/@okta/i18n/src/properties/login.properties

```diff
@@ -828,6 +828,15 @@ oie.on_prem.enroll.passcode.label = Enter authenticator passcode
 oie.on_prem.verify.title = Verify with {0}
 oie.on_prem.verify.passcode.label = Enter code
 
+## Custom Factor
+# {0} refers to to name of an authenticator for all Custom Factor values
+oie.customFactor.authenticator.description=Redirect to verify with {0}
+oie.customFactor.enroll.title=Set up {0}
+oie.customFactor.enroll.description=Clicking below will redirect to enrollment in {0}
+oie.customFactor.enroll.save=Enroll
```

Comment:
nit:
`enroll = Enroll` ?

Reply:
We don't have this property yet, but I've added a new one so future factors can reuse it!